### DISCUSSION TITLE
关于GEO多数据集联合的问题

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#54] [Q&A] [answered]
翔哥好，我最近看到文章有人把7个GEO数据集联合起来分析的，他说用combat去除了批次效应（Heterogeneity and potential variables are commonly recognized as major sources of bias and variability in high-throughput experiments. Since the datasets we recruited for our multi-datasets analysis were based on different platforms and samples were handled on different days, in different groups or by different people.），想问你这样的做法是否可行？不同平台都可以联合的话那岂不是我只需要把感兴趣的数据集联合成大的数据集，这个数据集在我研究方向上就万能了吗？
原文链接：https://www.nature.com/articles/s41598-018-34160-w
同时我还想到了xena上的TCGA_GTEX_TARGET数据集，他们好像不是简单的combat去除批次效应的，是不是证明这样去除批次效应其实是不科学的？
谢谢翔哥

**Comment by ShixiangWang** [comment, marked as answer]
combat去除批次效应是可行的，不去除批次效应肯定不行，至少要做这样的处理。批次效应本身就是为了去除不同平台、技术等导致的差异（即不是生物学本身的差异）。你可以联合多个平台进行分析，但一般而言，选择相同的芯片平台的数据比较好。使用的平台越多，数据集越多，你找到的差异不是实际差异的情况概率越高，即假阳性在上升。具体如何评估我就不清楚了。

后面Xena提到的数据集相当于从源头解决批次效应的问题，所以的处理手段都完全一致，自然没有批处理效应了。

**Comment by ShixiangWang** [comment]
有问题再交流，我先关了。
